### PR TITLE
Create anaconda-live sub package

### PR DESCRIPTION
**Pull request by M4rtinK**
The first commit creates the anaconda-live sub package & does related cleanups.

The second commit moves some GUI related files we forgot in anaconda-core to anaconda-gui.

### CODE REVIEW

File: anaconda.spec.in [hunk 1]
```diff
@@ -264,28 +270,24 @@ find %{buildroot} -type f -name "*.la" | xargs %{__rm}
 # Create an empty directory for addons
 mkdir %{buildroot}%{_datadir}/anaconda/addons
 
-%ifarch %livearches
+# needed for live installations
 desktop-file-install --dir=%{buildroot}%{_datadir}/applications %{buildroot}%{_datadir}/applications/liveinst.desktop
-%endif
 # NOTE: If you see "error: Installed (but unpackaged) file(s) found" that include liveinst files,
 #       check the IS_LIVEINST_ARCH in configure.ac to make sure your architecture is properly defined
```
Comment (poncovka) [inline anaconda.spec.in]: I guess that you can remove this note.

File: anaconda.spec.in [hunk 2]
```diff
@@ -149,6 +146,18 @@ Obsoletes: booty <= 0.107-1
 The anaconda-core package contains the program which was used to install your
 system.
 
+%package live
```
Comment (poncovka) [inline anaconda.spec.in]: Have you tried to create a live iso with this package?

File: anaconda.spec.in [hunk 3]
```diff
@@ -317,18 +324,23 @@ update-desktop-database &> /dev/null || :
...
+%{_datadir}/anaconda/gnome
 %{_sysconfdir}/xdg/autostart/*.desktop
-%endif
 
 %files gui
 %{python3_sitearch}/pyanaconda/ui/gui/*
+%{_datadir}/anaconda/pixmaps
+%{_datadir}/anaconda/ui
+%{_datadir}/anaconda/window-manager
+%{_datadir}/anaconda/anaconda-gtk.css
```
Comment (poncovka) [inline anaconda.spec.in]: Have you tried to run anaconda without anaconda-gui?

File: anaconda.spec.in (hunk 2 above)
Comment (jkonecny12) [reply anaconda.spec.in]: Also the opposite is required if the boot.iso installation will work without this package.

File: anaconda.spec.in [hunk 4]
```diff
@@ -149,6 +146,18 @@ Obsoletes: booty <= 0.107-1
...
+%package live
+Summary: Live installation specific files and dependencies
+BuildRequires: desktop-file-utils
+# live installation currently implies a graphical installation
+Requires: anaconda-gui = %{version}-%{release}
+Requires: usermode
+Requires: zenity
+
+%description live
+The anaconda-live package contains scripts, data and dependencies needed for
```
Comment (jkonecny12) [inline anaconda.spec.in]: Instead of `needed` the `required by` word sounds better to me.

File: anaconda.spec.in [hunk 5]
```diff
@@ -264,28 +270,24 @@ find %{buildroot} -type f -name "*.la" | xargs %{__rm}
 # Create an empty directory for addons
 mkdir %{buildroot}%{_datadir}/anaconda/addons
 
-%ifarch %livearches
+# needed for live installations
```
Comment (jkonecny12) [inline anaconda.spec.in]: Again required sounds better to me.

File: anaconda.spec.in (hunk 1 above)
Comment (M4rtinK) [reply anaconda.spec.in]: Removed.

File: anaconda.spec.in (hunk 2 above)
Comment (M4rtinK) [reply anaconda.spec.in]: Yep, I think we should create both a live image and boot.iso and check if both work. Hopefully we should have enough know-how to pull it off. ;-)

File: anaconda.spec.in (hunk 3 above)
Comment (M4rtinK) [reply anaconda.spec.in]: I think I should be able to try that on my Initial Setup VM & possibly we could try generating a text only boot.iso.

File: anaconda.spec.in (hunk 4 above)
Comment (M4rtinK) [reply anaconda.spec.in]: Changed.

File: anaconda.spec.in (hunk 5 above)
Comment (M4rtinK) [reply anaconda.spec.in]: Changed.

**Comment by M4rtinK** [comment]
Custom live iso created with the anaconda-live package split seems to work fine.

**Comment by M4rtinK** [comment]
A Boot iso created with the split live subpackage also works fine, so this PR is now ready to be merged.

**Pull request merged**